bladex/customprofile.py

```diff
@@ -45,67 +45,75 @@ class CustomProfile(ProfileBase):
     :param numpy.ndarray thickness_max: 1-element array expressing the maximum thickness 
        at a certain airfoil section
     """
-
     def __init__(self, **kwargs):
         super(CustomProfile, self).__init__()
-        
+
         if len(kwargs) == 4:
             xup, yup, xdown, ydown = kwargs.values()
             self.xup_coordinates = xup
             self.yup_coordinates = yup
             self.xdown_coordinates = xdown
             self.ydown_coordinates = ydown
-           
+            self.chord_len = 1
```

Comment:
This line can lead to several issues since the order of `kwargs` argument is assumed to be the "canonical".
Modfiy to
```
self.xup_coordinates = kwargs['xup']
...
```